modules/web/src/app/node-data/basic/provider/kubevirt/component.ts

```diff
@@ -186,7 +186,7 @@ export class KubeVirtBasicNodeDataComponent
       [Controls.Memory]: this._builder.control(this._defaultMemory, Validators.required),
       [Controls.PrimaryDiskOSImage]: this._builder.control('', Validators.required),
       [Controls.PrimaryDiskStorageClassName]: this._builder.control('', Validators.required),
-      [Controls.PrimaryDiskSize]: this._builder.control('10', Validators.required),
+      [Controls.PrimaryDiskSize]: this._builder.control('10G', Validators.required),
```

Comment:
We use `km-number-stepper` for this so I think `10G` value might break it. Can you double check that it works?

Reply:
Tested that locally and it seems to work fine.